httpx/decorator_test.go

```diff
@@ -133,12 +133,12 @@ func TestCheckHeaderDecorator(t *testing.T) {
 	assert.Equal(string(content), "")
 }
 
-func TestCheckSPAuthHeaderDecorator(t *testing.T) {
+func TestCheckAuthHeaderDecorator(t *testing.T) {
```

Comment:
test is duplicated (TestCheckHeaderDecorator)

Reply:
nice catch